tests/console/validate_modify_existing_partition.pm

```diff
@@ -16,12 +16,16 @@ use base "opensusebasetest";
 use testapi;
 use scheduler 'get_test_suite_data';
 use Test::Assert ':all';
+use validate_partitions;
 
 sub run {
     my $test_data = get_test_suite_data();
 
     select_console "root-console";
 
+    my $root_disk = {root_disk => "/dev/$test_data->{root}->{disk}"};
```

Comment:
I would make it explicit in test_data "/dev/<name>" so we don't need to mix hardcoded stuff with test data, but `/dev/` is not supposed to change, so up to you.